### PR TITLE
WIP: Redesign of consent screen

### PR DESCRIPTION
**Pull request by jorissteyn**
TODO:

 - [ ] Slide-ins as described in pivotal
 - [ ] Fix failing SSO behat test
 - [ ] Show logo for aggregation sources (currently showing a placeholder)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,18 @@ In short, themes require compilation which can be done by running the following
     (cd theme && npm install && grunt)
 ```
 
+Do setup the required tooling on the VM, the following steps might be useful:
```
Comment (MKodde) [inline README.md]: ~~Do~~ To

File: theme/material/javascripts/modules/ConsentPage.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
 
+    function initTooltips() {
+        tippy(
+            document.querySelectorAll('a.tooltip'),
+            {
+                animation: 'scale',
+                arrow: true,
+                duration: 200,
+                placement: 'top',
+                theme: 'light bordered'
```
Comment (MKodde) [inline theme/material/javascripts/modules/ConsentPage.js]: consider adding the `trigger: mouseenter` configuration option to keep mobile users happy.

File: languages/en.php [hunk 3]
```diff
@@ -173,7 +173,9 @@
     'consent_buttons_title'              => 'Do you agree with sharing this data?',
     'consent_buttons_ok'                 => 'Yes, proceed to %s',
     'consent_buttons_nok'                => 'No, I do not agree',
-    'consent_footer_text'                => '<a href="#">View or modify earlier approvals</a>',
+    'consent_footer_text'                => 'You have granted %1$s services access to your information.',
```
Comment (MKodde) [inline languages/en.php]: Would it be possible to pluralize this translation? When the user has granted it's consent to one service, this text would be incorrect grammatically.

Comment (MKodde) [inline languages/en.php]: For some odd reason the comment above was not added to the review..

Comment (jorissteyn) [reply languages/en.php]: Done, see new branch.

**Comment by jorissteyn** [comment]
Closing, see final version in PR #458.

**Pull request closed without merging**